src/components/Page/components/Header/Header.tsx

```diff
@@ -95,20 +111,39 @@ export function Header({
     (primaryAction.primary === undefined ? true : primaryAction.primary);
 
   const primaryActionMarkup = primaryAction ? (
-    <div className={styles.PrimaryActionWrapper}>
-      {buttonsFrom(primaryAction, {primary})}
-    </div>
+    <ConditionalWrapper
```

Comment:
Besides for a new render block, most of these changes in page are to remove the custom style wrappers. The new page is greatly simplified—it has a single container which is orthogonal to the contents. And the contents are primitive components and subcomponents, again, without custom wrappers.